dlib/cmake_utils/find_blas.cmake

```diff
@@ -50,7 +55,11 @@ if (UNIX OR MINGW)
 
       # We need cblas since MATLAB doesn't provide cblas symbols.
       add_subdirectory(external/cblas)
-      set(blas_libraries  cblas  )
+      if (NOT HAVE_CBLAS)
+         set(blas_libraries cblas blas)
+      else()
+         set(blas_libraries  cblas)
+      endif()
```

Comment:
This is wrong. `add_subdirectory` is creating cblas symbols.  Don't add these lines.  More generally, please do not add code that you are not testing.

Reply:
Okay, will remove.